### PR TITLE
DACT-679 AP01 submission confirmation page

### PR DESCRIPTION
**Pull request by mdesaikns**
https://companieshouse.atlassian.net/browse/DACT-679

### CODE REVIEW

File: views/appoint-director-submitted.html [hunk 1]
```diff
@@ -17,9 +17,63 @@
...
+        </div>
 
+        {{ govukSummaryList({
+          rows: [
+            {
+              key: {
+                text: "Company name"
+              },
+              value: {
+                text: companyName
```
Comment (olaych) [inline views/appoint-director-submitted.html:36]: Do we need to make sure this is always uppercase?

File: src/controllers/appoint.director.submitted.controller.ts [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
+    const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
+    const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
+    const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
+    const session: Session = req.session as Session;
+    const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
+    const officerFiling: OfficerFiling = await getOfficerFiling(session, transactionId, submissionId);
+
     return res.render(Templates.APPOINT_DIRECTOR_SUBMITTED, {
```
Comment (olaych) [inline src/controllers/appoint.director.submitted.controller.ts:23]: I can see back link on the prototype but not here. I suppose it was agreed to remove?

**Review by olaych — COMMENTED** [review]
Minor comments

File: views/appoint-director-submitted.html (hunk 1 above)
Comment (mdesaikns) [reply views/appoint-director-submitted.html:36]: I believe company name is always stored in capitalised cases, also this implementation is identical to TM01 confirmation page.

File: src/controllers/appoint.director.submitted.controller.ts (hunk 2 above)
Comment (mdesaikns) [reply src/controllers/appoint.director.submitted.controller.ts:23]: The prototype does not align with AC written, after discussing with the Product team we went with the decision to match the TM01 implementation and ACs written with the old prototype.  
The AC and TM01 don't have a back button hence I have not implemented it. 

Any new changes if they may come will be addressed in the snag tickets. 
https://companieshouse.atlassian.net/browse/DACT-679?focusedCommentId=226929

**Review by olaych — APPROVED** [review]

**Review by clewis1 — APPROVED** [review]
Looks good.

**Pull request merged**